README.md

```diff
@@ -22,13 +22,14 @@ Documentation is [in the doc tree](doc/), and can be compiled using [bookdown](h
 
 ```console
 $ bookdown doc/bookdown.json
-$ php -S 0.0.0.0:8080 -t doc/html/ # then browse to http://localhost:8080/
+$ php -S 127.0.0.1:8080 -t doc/html/ # then browse to http://localhost:8080/
```

Comment:
The above was correct. `0.0.0.0` indicates it should listen on any available IP served by the machine, which will also always include 127.0.0.1.